Declarations/Constants.ts

```diff
@@ -185,4 +184,9 @@ export enum StatsbeatInstrumentation {
     BUNYAN = 32,
     WINSTON = 64,
     CONSOLE = 128,
-}
\ No newline at end of file
+}
+
+export enum StatsbeatFeatureType {
```

Comment:
FeatureStatsbeatType?

Reply:
All types and enums in this file start with Statsbeat word to quickly find them